### ISSUE TITLE
[BUG] Device loss detected in Present()

### ISSUE THREAD

**Issue by ttiee** [jeertmans/manim-slides#315]
### Description

The output:
```powershell
(.env-3.8.10) E:\Manim\manimgl-manim-slides>manim-slides BasicExample
    failed to get textures for frame; format: 172 textureConverter null
Device loss detected in Present()
Device loss detected in Present()
```
And the Screeen is frozen

### Version

`Manim Slides v5.0.0`

### Platform

`Win11`

### Screenshots

<img width="1280" alt="image" src="https://github.com/jeertmans/manim-slides/assets/98325911/c6ef1ec7-eaec-409e-b11b-4bc30ea307cc">

<img width="1275" alt="image" src="https://github.com/jeertmans/manim-slides/assets/98325911/5f018694-0c1e-435c-a59c-0dd61b3adbfe">

**Comment by ttiee** [comment]
here is the output, when setting `QT_DEBUG_PLUGINS=1`
The output is too long:
```powershell
(.env-3.8.10) E:\Manim\manimgl-manim-slides>manim-slides NextSlideExample
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms" ...
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qdirect2d.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qdirect2d.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.3",
    "MetaData": {
        "Keys": [
            "direct2d"
        ]
    },
    "archlevel": 1,
    "className": "QWindowsDirect2DIntegrationPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("direct2d")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qminimal.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qminimal.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.3",
    "MetaData": {
        "Keys": [
            "minimal"
        ]
    },
    "archlevel": 0,
    "className": "QMinimalIntegrationPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("minimal")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qoffscreen.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qoffscreen.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.3",
    "MetaData": {
        "Keys": [
            "offscreen"
        ]
    },
    "archlevel": 0,
    "className": "QOffscreenIntegrationPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("offscreen")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qwindows.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qwindows.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPA.QPlatformIntegrationFactoryInterface.5.3",
    "MetaData": {
        "Keys": [
            "windows"
        ]
    },
    "archlevel": 0,
    "className": "QWindowsIntegrationPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("windows")
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/platforms" ...
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platforms/qwindows.dll" loaded library
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/platformthemes" ...
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/platformthemes" ...
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/styles" ...
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/styles/qwindowsvistastyle.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/styles/qwindowsvistastyle.dll, metadata=
{
    "IID": "org.qt-project.Qt.QStyleFactoryInterface",
    "MetaData": {
        "Keys": [
            "windowsvista"
        ]
    },
    "archlevel": 0,
    "className": "QWindowsVistaStylePlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("windowsvista")
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/styles" ...
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/styles/qwindowsvistastyle.dll" loaded library
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/iconengines" ...
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/iconengines/qsvgicon.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/iconengines/qsvgicon.dll, metadata=
{
    "IID": "org.qt-project.Qt.QIconEngineFactoryInterface",
    "MetaData": {
        "Keys": [
            "svg",
            "svgz",
            "svg.gz"
        ]
    },
    "archlevel": 0,
    "className": "QSvgIconPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("svg", "svgz", "svg.gz")
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/iconengines" ...
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats" ...
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qgif.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qgif.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "gif"
        ],
        "MimeTypes": [
            "image/gif"
        ]
    },
    "archlevel": 0,
    "className": "QGifPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("gif")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qicns.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qicns.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "icns"
        ],
        "MimeTypes": [
            "image/x-icns"
        ]
    },
    "archlevel": 0,
    "className": "QICNSPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("icns")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qico.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qico.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "ico",
            "cur"
        ],
        "MimeTypes": [
            "image/vnd.microsoft.icon",
            "image/vnd.microsoft.icon"
        ]
    },
    "archlevel": 0,
    "className": "QICOPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("ico", "cur")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qjpeg.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qjpeg.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "jpg",
            "jpeg"
        ],
        "MimeTypes": [
            "image/jpeg",
            "image/jpeg"
        ]
    },
    "archlevel": 0,
    "className": "QJpegPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("jpg", "jpeg")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qpdf.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qpdf.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "pdf"
        ],
        "MimeTypes": [
            "application/pdf"
        ]
    },
    "archlevel": 0,
    "className": "QPdfPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("pdf")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qsvg.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qsvg.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "svg",
            "svgz"
        ],
        "MimeTypes": [
            "image/svg+xml",
            "image/svg+xml-compressed"
        ]
    },
    "archlevel": 0,
    "className": "QSvgPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("svg", "svgz")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtga.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtga.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "tga"
        ],
        "MimeTypes": [
            "image/x-tga"
        ]
    },
    "archlevel": 0,
    "className": "QTgaPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("tga")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtiff.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtiff.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "tiff",
            "tif"
        ],
        "MimeTypes": [
            "image/tiff",
            "image/tiff"
        ]
    },
    "archlevel": 0,
    "className": "QTiffPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("tiff", "tif")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwbmp.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwbmp.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "wbmp"
        ],
        "MimeTypes": [
            "image/vnd.wap.wbmp"
        ]
    },
    "archlevel": 0,
    "className": "QWbmpPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("wbmp")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwebp.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwebp.dll, metadata=
{
    "IID": "org.qt-project.Qt.QImageIOHandlerFactoryInterface",
    "MetaData": {
        "Keys": [
            "webp"
        ],
        "MimeTypes": [
            "image/webp"
        ]
    },
    "archlevel": 0,
    "className": "QWebpPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("webp")
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/imageformats" ...
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qgif.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qicns.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qico.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qjpeg.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qpdf.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qsvg.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtga.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qtiff.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwbmp.dll" loaded library
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/imageformats/qwebp.dll" loaded library
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia" ...
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia/ffmpegmediaplugin.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia/ffmpegmediaplugin.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPlatformMediaPlugin",
    "MetaData": {
        "Keys": [
            "ffmpeg"
        ]
    },
    "archlevel": 0,
    "className": "QFFmpegMediaPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("ffmpeg")
qt.core.plugin.factoryloader: looking at "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia/windowsmediaplugin.dll"
qt.core.plugin.loader: Found metadata in lib E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia/windowsmediaplugin.dll, metadata=
{
    "IID": "org.qt-project.Qt.QPlatformMediaPlugin",
    "MetaData": {
        "Keys": [
            "windows"
        ]
    },
    "archlevel": 0,
    "className": "QWindowsMediaPlugin",
    "debug": false,
    "version": 394496
}


qt.core.plugin.factoryloader: Got keys from plugin meta data QList("windows")
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/multimedia" ...
qt.core.library: "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/multimedia/ffmpegmediaplugin.dll" loaded library
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/accessible" ...
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/accessible" ...
qt.core.plugin.factoryloader: checking directory path "E:/manim/manimgl-manim-slides/.env-3.8.10/lib/site-packages/PySide6/plugins/accessiblebridge" ...
qt.core.plugin.factoryloader: checking directory path "E:/python3.8.10/accessiblebridge" ...
    failed to get textures for frame; format: 172 textureConverter null
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
Device loss detected in Present()
```

**Comment by jeertmans** [comment]
Thanks for creating a new issue @ttiee!
Can you open the individual video files in the slides folder with any video application? Like Windows Media Player or VLC?

also, does the error message occur before playing any of the slide?

**Comment by ttiee** [comment]
The video can be played nomally.
<img width="1280" alt="image" src="https://github.com/jeertmans/manim-slides/assets/98325911/c1ab3bf3-b71a-44e9-a66d-62356d153df4">
The error message occured at the beginning. As long as run `manim-slides`, error occures and the screen is frozen

**Comment by jeertmans** [comment]
Can you try running this example from PySide6's documentation and play one of the video files generated by `manim-slides`? This is a video player that uses the same base class to play video files, so I want to check if this works.

Link: https://doc.qt.io/qtforpython-6/examples/example_multimedia_player.html.

Also, can you show me what is the version of PySide6 you are using?

You can obtain the information with `pip show PySide6`.

**Comment by ttiee** [comment]
`pip show PySide6`
```powershell
(.env-3.8.10) E:\Manim\manimgl-manim-slides>pip show PySide6
Name: PySide6
Version: 6.5.2
Summary: Python bindings for the Qt cross-platform application and UI framework
Home-page: https://www.pyside.org
Author: Qt for Python Team
Author-email: pyside@qt-project.org
License: LGPL
Location: e:\manim\manimgl-manim-slides\.env-3.8.10\lib\site-packages
Requires: PySide6-Addons, PySide6-Essentials, shiboken6
Required-by: manim-slides
```
OK, I will try the example from PySide6's documentation

**Comment by ttiee** [comment]
https://github.com/jeertmans/manim-slides/assets/98325911/2c0b9bb3-3ffe-4a9a-a9de-f94fd22d0996

I cut the video to make it small. The audio could be played, but the video can't.

**Comment by ttiee** [comment]
<img width="1280" alt="image" src="https://github.com/jeertmans/manim-slides/assets/98325911/2f402eee-137b-4e0f-9c37-7c52757a21ce">

I create a virtual environment python=3.11.6 PySide6=6.6.0, the example from PySide6's documentation works well, even the output alse contains the same warning
```powershell
failed to get textures for frame; format: 172 textureConverter null
```

**Comment by ttiee** [comment]
Ohhh! I runned the `pip install pyside6==6.6.0` tore-installed the pyside=6.6.0. It works well

<img width="1275" alt="屏幕截图 2023-11-19 224835" src="https://github.com/jeertmans/manim-slides/assets/98325911/1c93802f-fa20-43f3-9998-b9b4bc3488cc">

**Comment by jeertmans** [comment]
Oh ok so this is definitely a bug from PySide6.

The issue is that version 6.6.0 seems to contain bugs (which I reported to them), but maybe this is not the case: do you have Nice transitions between slides or do you observe flashing black screen in between?

**Comment by ttiee** [comment]
Yes, There are flashing black screens in between transitions.

The bug exists in the 6.5.2 version of pyside6, but in the 6.6.0 maybe pyside6 fixed some bug. So the screen isn't frozen. But the bug still exists.

**Comment by jeertmans** [comment]
> Yes, there are flashing black screens in between transitions.

This is actually what I discovered with #293.

> The bug exists in the `6.5.2` version of `pyside6`, but in the `6.6.0` maybe pyside6 fixed some bug. So the screen isn't frozen. But the bug still exists.

Do you mean that you also observe the flashing black screen in `6.5.2`? Or you are referring to the other bug (see this issue)?

**Comment by jeertmans** [comment]
Anyway, you can try to install another version of PySide6, version between `>=6.5.0` and `<6.5.3` should all work, hopefully.

**Comment by ttiee** [comment]
> > Yes, there are flashing black screens in between transitions.
> 
> This is actually what I discovered with #293.
> 
> > The bug exists in the `6.5.2` version of `pyside6`, but in the `6.6.0` maybe pyside6 fixed some bug. So the screen isn't frozen. But the bug still exists.
> 
> Do you mean that you also observe the flashing black screen in `6.5.2`? Or you are referring to the other bug (see this issue)?

What I mean is that in the PySide6 version 6.6.0, at least it can still play the video, but in version 6.5.2, it cannot play the video at all—upon running, it freezes.

> Anyway, you can try to install another version of PySide6, version between `>=6.5.0` and `<6.5.3` should all work, hopefully.

On my computer, video playback functions properly only with versions >= 6.5.3; however, there is a brief black screen when switching between videos.

**Comment by jeertmans** [comment]
Thanks for the details! What version of Windows are you using? I might try on my own Windows computer when I have time :-)

**Comment by jeertmans** [comment]
Hello @logdog,
I am sorry to hear that you are facing issues using this tool!

Regarding the first part of your message, you can see that you are using PySide6.5.0, but version 6.5.1 or 6.5.2 should probably fix your issue.

If not, then did you Google your graphics card issue? I am not aware of such issue.. 

Starting 6.5.3, Qt has a bug (flashing black screens) and they have not patched it yet :/
I did not try the new 6.6.2, but I doubt they fixed it because my issue is still open on their big tracker…

**Comment by logdog** [comment]
Update: I resolved my problem by rolling back `manim-slides=5.0.0`  and using `pyside6==6.5.2`

Let me try some other version of manim-slides and see which ones work.

Edit: version 5.1.0 - 5.1.3 all fail on my computer. Version 5.0.0 is the only one that works for me (I did not try any of the version 5.1 pre-releases)

**Comment by logdog** [comment]
Yes, similar error message when running with `manim-slides==5.1.3`. Here is the output:

```
(env) C:\Users\logan\OneDrive\Documents\YouTube\Private\Cart Pole>manim-slides              
1: Lagrangian
2: Test

Choose number corresponding to desired scene/arguments.
(Use comma separated list for multiple entries)
Choice(s): 2
    failed to get textures for frame; format: 172 textureConverter null
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975354632368
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975354632368
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372485488
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372485488
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372488688
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372488688
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372489456
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372489456
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372480688
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975372480688
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405395056
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405395056
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405402288
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405402288
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405391856
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405391856
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405387056
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405387056
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405388656
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405388656
Device loss detected in Present()
Failed to create srv: COM error 0x887a0005: The GPU device instance has been suspended. Use GetDeviceRemovedReason to determine the appropriate action.
Failed to initialize QRhiTexture wrapper for native texture object 1975405387888
```

**Comment by jeertmans** [comment]
Thanks, I really need to open Windows back to check this issue 🥲

**Comment by logdog** [comment]
I have confirmed this is an issue with my graphics card. I disabled my graphics driver (Intel(R) Iris(R) Xe Graphics 31.0.101.3688) and `manim-slides` worked perfectly now. However, this makes it impossible to use multiple monitors, so it is not a long term solution.

I will continue to just use `manim-slides==5.0.0` and `pyside6==6.5.2` for the time being.

**Comment by jeertmans** [comment]
Thanks for the report @logdog! Does it work with more recent versions of Manim Slides though?
